src/schema/search.ts

```diff
@@ -63,6 +92,18 @@ export const typeDefs = /* GraphQL */ `
 
 export const resolvers: IResolvers<unknown, Context> = traceResolvers({
   Query: {
+    searchSessionHistory: async (
+      _,
+      { limit, lastId }: GQLSearchSessionParams,
+      ctx,
+    ): Promise<ConnectionRelay<GQLSearchSession>> => {
+      return graphorm.queryPaginatedIntegration(
+        () => !!lastId,
+        (nodeSize) => nodeSize === limit,
+        (node) => base64(`timestamp:${node.createdAt}`),
```

Comment:
In our case the cursor should be created by the id as it's the property that we pass for the pagination

Reply:
Let's have a session when you come back from your holiday and I'll explain :)

Reply:
Amazing. Looking forward to it! 🚀